compiler/src/typed/parmatch.re

```diff
@@ -1972,7 +1961,7 @@ let check_unused = (pred, casel) =>
     let rec do_rec = pref =>
       fun
       | [] => ()
-      | [{mb_pat: q, /*c_guard;*/ mb_body}, ...rem] => {
+      | [{mb_pat: q, mb_guard, mb_body}, ...rem] => {
```

Comment:
It's so cool when some piece of commented code gets uncommented to add a killer feature! 🎉